eZ/Bundle/EzPublishCoreBundle/Tests/Imagine/VariationPathGenerator/WebpFormatVariationPathGeneratorTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Tests\Imagine\VariationPathGenerator;
```

Comment:
the same for test classes - they need to be in `tests/` in a corresponding namespace.